src/release/process.md

```diff
@@ -70,22 +75,14 @@ git push git@github.com:rust-lang/rust $BRANCH_POINT:beta -f
 ```
```

Comment:
Can't comment directly but there's a mention of "yesterday" on line 71.

Reply:
Fixed.